addons/l10n_es/data/template/account.account-es_pymes.csv

```diff
@@ -4,11 +4,15 @@
 "account_pymes_1034","Socios por desembolsos no exigidos, capital pendiente de inscripción","1034","liability_non_current"
 "account_pymes_1040","Socios por aportaciones no dinerarias pendientes, capital social","1040","liability_non_current"
 "account_pymes_1044","Socios por aportaciones no dinerarias pendientes, capital pendiente de inscripción","1044","liability_non_current"
+"account_common_112","Reserva legal","112","liability_non_current"
```

Comment:
@william-andre @oco-odoo New CoA or this hack keeping the old xml_id?

Reply:
Not sure I understand, but if you are reloading the template and an xmlid is not found, it will try searching on the code instead